tests/src/Interop/MarshalAPI/String/StringMarshalingTest.cs

```diff
@@ -52,6 +52,28 @@ private void StringToBStrToString()
         }
     }
 
+    private unsafe void SecureStringToBSTRToString()
+    {
+        foreach (String ts in TestStrings)
+        {
+            SecureString secureString = new SecureString();
+            foreach (char character in ts)
+            {
+                secureString.AppendChar(character);
+            }
+
+            IntPtr BStr = Marshal.SecureStringToBSTR(secureString);
+            String str = Marshal.PtrToStringBSTR(BStr);
+
+            if (!str.Equals(ts))
```

Comment:
just Assert.Equals(str, ts) ? that dumps the strings in the test log if they differ.